samples/sensor/stm32_vbat_sensor/boards/nucleo_g071rb.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for creating vbat sensor device instance
+ */
+
+/ {
+	stm-vbat {
+		compatible = "st,stm32-vbat";
+		ratio = <3>;
+		io-channels = <&adc1 14>;
+		status = "okay";
+		label = "VBAT";
+	};
+};
+
+&adc1 {
+    status = "okay";
+    has-vbat-channel;
```

Comment:
Please populate `has-vbat-channel` directly in dtsi files similarly to what was done for `has-temp-channel` and `has-vref-channel`

Reply:
ok